lib/Number/Compare.pm

```diff
@@ -1,8 +1,10 @@
 package Number::Compare;
 use strict;
+use warnings;
+
 use Carp qw(croak);
-use vars qw/$VERSION/;
-$VERSION = '0.03';
+
+our $VERSION = '0.04';
```

Comment:
Can you change this over to `our $VERSION = '0.03';`, I prefer to concrete in the new version as part of my release prep.

Reply:
Ok!